components/clock.rst

```diff
@@ -147,3 +147,61 @@ being in a month or another.
 .. versionadded:: 6.3
 
     The :class:`Symfony\\Component\\Clock\\ClockAwareTrait` was introduced in Symfony 6.3.
+
+Writing time-sensitive tests
```

Comment:
```suggestion
Writing time-sensitive Tests
```

Reply:
Updated! Is there a rule to only capitalize "Tests" but not "time-sensitive"? 🙂

Reply:
We are using, what is called title case, but to be honest, I am unsure about the capitalization of "time-sensitive", maybe @javiereguiluz can help us out here.

And maybe this could be a helpful addition for the contribution guide for the docs 🤷

Reply:
If Javier have a clear idea about this topic, I'll be more than happy to add it to the contribution guide! 👍

Reply:
The title-case is mentioned in https://symfony.com/doc/current/contributing/documentation/standards.html#english-language-standards

About hyphenated words:

> As for whether hyphenated words are capitalised not only at the beginning but also after the hyphen, there is no universal standard; variation occurs [in the wild](https://en.wikipedia.org/wiki/Linguistic_description) and among house styles (e.g., "The Letter-Case Rule in My Book"; "Short-term Follow-up Care for Burns").

So, maybe it's correct to write it as `Writing Time-Sensitive Tests`